features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -680,7 +680,7 @@
   "console.session.params.sessionRefreshTimeOut": 300,
   "console.session.params.checkSessionInterval": 3,
   "console.applications.enabled": true,
-  "console.product_version.configs.productVersion": "BETA",
+  "console.product_version.configs.productVersion": "",
```

Comment:
do we need this config here? we can remove it from the default.json file

Reply:
This commit removes it https://github.com/wso2/carbon-identity-framework/pull/4129/commits/d1160f2b108a0610d09266b80ec89044d1f4a3ba